nncf/quantization/layers.py

```diff
@@ -257,8 +257,6 @@ def calculate_level_ranges(num_bits, signed, is_weights):
             level_high = 2 ** num_bits - 1
             level_low = 0
         levels = 2 ** num_bits
-        if is_weights:
```

Comment:
It's strange that pre-commit is OK, while reference implementation in tests still has 255 levels. 

At least it should be also corrected in `test_functions.py`

Reply:
I suppose the tests were passed as in **test_functions.py** we utilize the same levels-255 for reference and symmetric_quantize.